network/builder/src/builder.rs

```diff
@@ -221,13 +217,18 @@ impl NetworkBuilder {
             || !config.seeds.is_empty()
         {
             let mut seeds = config.seeds.clone();
+
+            // Merge old seed configuration with new seed configuration
+            // TODO(gnazario): Once fully migrated, remove `seed_addrs`
             config
                 .seed_addrs
                 .iter()
-                .map(|(peer_id, addrs)| (peer_id, TrustedPeer::from(addrs.clone())))
+                .map(|(peer_id, addrs)| {
+                    (peer_id, Peer::from_addrs(PeerRole::Upstream, addrs.clone()))
```

Comment:
Oh boy, What's the plan here? I guess this is cause we don't have good handles on our testnet repo versus our release repo?

Reply:
The plan here is once we get through all users of the old `peer_addrs` field, we can just axe this right out.  When we have a little bit more stable planning between the testnet vs release, we'll be good to get rid of this.